### PR TITLE
Przemek database

### PR DESCRIPTION
**Pull request by Isild**
Database for graph.
Text analyse create new graph, return it id.
Endpoint to get graph data by id.

### CODE REVIEW

File: server/app/services/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+from app.services.db import db
```
Comment (wojtasiq) [inline server/app/services/__init__.py:2]: ?

File: server/.gitignore [hunk 2]
```diff
@@ -217,3 +217,5 @@ venv.bak/
 pip-selfcheck.json
 
 # End of https://www.gitignore.io/api/python,virtualenv
+
+*.db
```
Comment (wojtasiq) [inline server/.gitignore:221]: add line at the end of file please

File: server/app/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
+
```
Comment (wojtasiq) [inline server/app/models/__init__.py:2]: remove?

**Comment by Isild** [comment]
please ignore Lint failed tests, it corrupts applications and make circular import in database.py when import Graph model

**Comment by lpawelcz** [comment]
> please ignore Lint failed tests, it corrupts applications and make circular import in database.py when import Graph model

ok, but those long lines in `graph.py` should be shortened

**Pull request merged**